### PR TITLE
Add March21 release note

### PR DESCRIPTION
**Pull request by allandenis38**
Thank you for your contribution to the Axway Streams documentation.

## Describe the changes

Add March21 release note 

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links

### CODE REVIEW

**Comment by allandenis38** [comment]
Hi @lbadenhop this MR is ready for review. Thx.

File: content/en/docs/relnotes/20210331_streams_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+| SSE Subscriber | /streams/subscribers/sse | /streams/subscribers/sse/api/v1 |
+
+#### Pagination
+
+Links related to [pagination](/docs/topics-api/#Pagination) are now provided as part of the response body.
+
+#### Json Object as root element of the response body
+
+Streams Rest APIs no longer return an `array` as root element of the response.
+APIs returning a list of elements now return a JSON Objet as root element and an `array` wrapped in `items` attribute.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210331_streams_relnotes.md:44]:
```suggestion
APIs returning a list of elements now return a JSON Object as root element and an `array` wrapped in `items` attribute.
```

File: content/en/docs/relnotes/20210331_streams_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ To stay current and align our offerings with customer demand and best practices, Axway might discontinue support for some capabilities.
+
+Following capabilities related to Streams Rest APIs are no longer supported:
+
+* Old Rest API Paths, refer to [New Rest API Paths](/docs/relnotes/20210331_streams_relnotes/#new-rest-api-paths).
+* Pagination related information provided as `Link` Header is no longer provided, refer to [Pagination](/docs/relnotes/20210331_streams_relnotes/#pagination).
+
+## Fixed issues
+
+There are no fixed issue in this version.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210331_streams_relnotes.md:80]:
```suggestion
There are no fixed issues in this version.
```

File: content/en/docs/relnotes/20210331_streams_relnotes.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+You can find the latest information and up-to-date user guides at the Axway Documentation portal at <https://docs.axway.com>.
+
+This section describes documentation enhancements and related documentation.
+
+### Documentation enhancements
+
+<!-- Add a summary of doc changes or enhancements here-->
+
+The latest version of Streams documentation has been migrated to Markdown format and is available in a [public GitHub repository](https://github.com/Axway/streams-open-docs) to prepare for future collaboration using an open source model. As part of this migration, the documentation has been restructured to help users navigate the content and find the information they are looking for more easily.
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210331_streams_relnotes.md:92]:
```suggestion
The latest version of Streams documentation has been migrated to Markdown format and is available in a [public GitHub repository](https://github.com/Axway/streams-open-docs) to prepare for future collaboration using an open-source model. As part of this migration, the documentation has been restructured to help users navigate the content and find the information they are looking for more easily.
```

File: content/en/docs/relnotes/20210331_streams_relnotes.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+* [Custom Metrics](/docs/metrics) to provide additional insights on the health of Streams components when monitored via Prometheus.
+
+## Important changes
+<!-- Use this section to describe any changes in the behavior of the product (as a result of features or fixes), for example, new Java system properties in the jvm.xml file. This section could also be used for any important information that doesn't fit elsewhere. -->
+
+It is important, especially when upgrading from an earlier version, to be aware of the following changes in the behavior or operation of the product in this new version.
+
+### Streams REST APIs
+
+Following change on Streams Rest APIs have been made to fully comply with Axway Rest API Guidelines:
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210331_streams_relnotes.md:25]:
```suggestion
The following changes have been have to the Streams Rest APIs to fully comply with Axway Rest API Guidelines:
```

File: content/en/docs/relnotes/20210331_streams_relnotes.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+<!-- Add features that are deprecated here -->
+
+As part of our software development life cycle, we constantly review our Streams offering.
+As part of this review, no capabilities have been deprecated.
+
+## Removed features
+<!-- Add features that are removed here -->
+ To stay current and align our offerings with customer demand and best practices, Axway might discontinue support for some capabilities.
+
+Following capabilities related to Streams Rest APIs are no longer supported:
```
Comment (lbadenhop) [inline content/en/docs/relnotes/20210331_streams_relnotes.md:73]:
```suggestion
The following capabilities related to Streams Rest APIs are no longer supported:
```

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**